src/constants.js

```diff
@@ -0,0 +1,230 @@
+const offerTypes = {
+  TAXI: 'Taxi',
+  BUS:'Bus',
+  TRAIN:'Train',
+  SHIP:'Ship',
+  DRIVE:'Drive',
+  FLIGHT:'Flight',
+  CHECKIN:'Check-in',
+  SIGHTSEENING:'Sightseeing',
+  RESTARAUNT:'Restaurant'
+};
+
+const OFFERS_TYPES = Object.values(offerTypes);
+
+const CITY = [
+  'Geneva',
+  'Chamonix',
+  'Amsterdam',
+  'Dublin',
+  'Lisbon',
+  'Berlin',
+];
+
+const EVENT_POINT_COUNT = 15;
+
+const additionalOffers = [
+  {
+    'type': 'Taxi',
```

Comment:
```suggestion
    'type': OfferType. TAXI,
```
Переиспользуй енам, не зря же мы его создавали 🙂